### PR TITLE
update readme for checking stack version

### PR DESCRIPTION
**Pull request by dividedbynil**
Added stack version check command.

### CODE REVIEW

**Comment by dividedbynil** [comment]
I haved signed the CLA.

**Comment by dividedbynil** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ fit xData yData = TF.runSession $ do
 # Installation Instructions
 
 Note: building this repository with `stack` requires version `1.4.0` or newer.
+Check your stack version with `stack --version` in terminal.
```
Comment (judah) [inline README.md]: Grammar nit: "in a terminal".  (I've edited the PR with that change.)

**Comment by judah** [comment]
Thank you for the patch!

**Comment by dividedbynil** [comment]
@judah You're welcome!

**Pull request merged**